onnxruntime/core/providers/tensorrt/common.cuh

```diff
@@ -0,0 +1,302 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
```

Comment:
This is all duplicate code that would need to ne maintained such as elementwise ops. If my understanding is correct, let's brainstorm on what causes the duplicates.,

Reply:
I was able to make it build from the cuda source files directly, so now there's only one copy. So no more duplicates!